README.md

```diff
@@ -8,6 +8,11 @@ lists all the _Cloud_ APIs you can call from .NET.
 
 The samples in this repo cover only _some_ of the total APIs that you can call from .NET.
 
+## Google Cloud Samples
+
+To browse ready to use code samples check [Google Cloud Samples](https://cloud.google.com/docs/samples).
```

Comment:
Most of the samples in sample browser are from these sample repos. This notes gives the impression as if the ones in sample browser were different. I believe a better message would be:
```suggestion
You can also browse these samples, and more in [Google Cloud Samples](https://cloud.google.com/docs/samples).
```